### ISSUE TITLE
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)

### ISSUE THREAD

**Issue by aureliusm** [tighten/takeout#121]
Hi, thanks for a great product!

After updating docker 2.3.0.5, I had to restart all services, but after restarting I now cannot login to my mysql.

Error I get `ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)`

I already tried null password, "password" as password etc. and nothing helps.

Any ideas? 

Thanks!

**Comment by mattstauffer** [comment]
A few questions:

1. Where are you seeing this error? Is this command line MySQL, or your PHP application?
2. Have you tried disabling the `mysql` service and re-enabling it so it's generated by the latest version of Takeout?

Thanks!

**Comment by aureliusm** [comment]
Thanks for a quick response.

1. I see it when running `docker exec -it e509c07994b1 mysql -p` or through "Sequel Ace" or via PHP application - all three worked before
2. I have tried it yes multiple times and also updated takeout to the latest version

**Comment by aureliusm** [comment]
Just letting you know that the issue was that my mysql_data volume has been created with almost the first version of takeout and so when you introduced this change `Implemented mysql_native_password native authentication plugin for all MySQL containers` it messed things up, because the old volume still had old info. 

Once I removed the mysql_data volume `docker volume rm mysql_data` (thankfully I didn't have anything important there), it all works again after enabling mysql via takeout.

**Comment by mattstauffer** [comment]
Ahhh.. that's helpful to know, I didn't know that would break things. Thanks!

**Comment by aureliusm** [comment]
@mattstauffer Just letting you know that I experienced this again. The only thing that happened was that my docker was updated so all my containers were stopped. I then had to `takeout enable mysql` again and it bumped the mysql version to 8.0.22 - all went fine. But when trying to connect via console `docker exec -it 2a831e763c7d mysql` (or otherwise), I again get the error `ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: NO)`.

Any ideas how to approach getting to the data in `mysql_data` volume?

Thanks!
